src/routes/SignUp.jsx

```diff
@@ -0,0 +1,50 @@
+import { getAuth, updateEmail, createUserWithEmailAndPassword } from 'firebase/auth';
+import React, { useState } from 'react';
+import { auth } from '../utils/firebase';
+
+const SignUp = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const auth = getAuth();
+
+  createUserWithEmailAndPassword(auth, email, password)
```

Comment:
move this function inside the signup. This function is called everytime email and password state get updated.